src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/WipStateChanged.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  The MIT License
+ *
+ *  Copyright 2017 Axis Communications AB. All rights reserved.
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package com.sonymobile.tools.gerrit.gerritevents.dto.events;
+
+import com.sonymobile.tools.gerrit.gerritevents.dto.GerritEventType;
+
+/**
+ * A DTO representation of the wip-state-changed Gerrit Event.
+ *
+ * @author Sven Selberg &lt;sven.selberg@axis.com&gt;
+ */
+public class WipStateChanged extends ChangeBasedEvent {
+  @Override
+  public GerritEventType getEventType() {
+    return GerritEventType.PRIVATE_STATE_CHANGED;
```

Comment:
@darfink  shouldn't this line be GerritEventType.WIP_STATE_CHANGED?

Reply:
It most certainly should, great catch!